Controllers/Mobile/v1/RatingsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AonFreelancing.Models.DTOs;
+using AonFreelancing.Services;
+using AonFreelancing.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace AonFreelancing.Controllers.Mobile.v1
+{
```

Comment:
Since all this controller's methods require authorization, you could place [Authorize] on top of the class, insead of repeating it over and over.

Reply:
Thank you for reminding me.